### PR TITLE
Roadblocks for ada assessment

### PR DESCRIPTION
**Pull request by DevChima**
Handle roadblock cardType sent from ADA
Also fix MQR test causing the build to fail.

### CODE REVIEW

File: ada/models.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ class AdaSelfAssessment(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     choice = models.TextField(null=True, blank=True)
     pdf_media_id = models.CharField(max_length=255, null=True, blank=True)
+    roadblock = models.TextField(null=True, blank=True)
```
Comment (erikh360) [inline ada/models.py]: Should this be a text field? looks like we only save ROADBLOCK in there?

**Review by erikh360 — COMMENTED** [review]
I'll fix the MQR test properly at a later stage

File: ada/models.py (hunk 1 above)
Comment (DevChima) [reply ada/models.py]: Yes @erikh360 . Initially wasn't sure what we would save there. I'll change to CharField

**Comment by Buhle79** [comment]
+1

**Pull request merged**